.gitignore

```diff
@@ -1,6 +1,7 @@
 /dist
 /.idea
 /.vscode
+/vendor
```

Comment:
I would suggest to remove this change from this PR, and discuss if it is correct to be set in another place.

Reply:
i can remove it.. but lets discuss and remove it. i don't understand why it is inside the code in the first place.

Reply:
We are not vendoring code in the bee project, just using modules. By having a vendor folder locally and ignored by git, it may present problems by using different code do run build or test, then the one defined in go.mod. If vendor directory is present in the project dir, go will use that code instead modules cache. This may result in different results when a developers runs code locally and on ci. If the vendor is not ignored it forces the developer to clean it if it is used for some experiment for example.